PIP_README.md

```diff
@@ -234,5 +232,17 @@ specifications. Below is a table of all the configurable inputs that can be used
     #[cbdefenseserver2]
     #api_connector_id = KJARWBZ111
     #api_key = CQF35EIH2WDF69PTWKGC4111
+    #siem_connector_id = UEUWR4U111
+    #siem_api_key = XNS5UKWZXZMCC3CYC7DFM111
```

Comment:
Are these our real API/SIEM keys?? Should we consider utilizing a fake one?

Reply:
No they are fake.

Reply:
Awesome, thanks for the check :)